pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +33,32 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlags adds output related flags to a command. Used by mutations only.
+func AddOutputFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use -o name for showing only kind/name.")
+}
+
+// PrintSuccess prints message after finishing mutating operations
+func PrintSuccess(cmd *cobra.Command, out io.Writer, resource string, name string, operation string) {
+	outputMode := GetFlagString(cmd, "output")
+	if outputMode == "name" {
+		// -o name: prints kind/name
+		fmt.Fprintf(out, "%s/%s\n", resource, name)
+	} else {
+		// understandable output by default
+		fmt.Fprintf(out, "%s \"%s\" %s successfully\n", resource, name, operation)
```

Comment:
I'm not entirely opposed to dropping "successfully", though one of the main points of this is to address user complaints that they couldn't tell whether operations were successful or not. Also, I could imagine adding a similar statement to prefix errors: `kind name <process of doing the command, e.g., creation> failed`

I prefer the verb after the resource, but don't have a strong opinion if we agree to drop "successfully".

We shouldn't put a slash in between the kind and name. That doesn't match the canonical format accepted by kubectl and isn't human-oriented.